apps/mobile/src/components/Login/WalletSelectorBottomSheet.tsx

```diff
@@ -0,0 +1,142 @@
+import { useBottomSheetDynamicSnapPoints } from '@gorhom/bottom-sheet';
+import clsx from 'clsx';
+import { ForwardedRef, forwardRef, useRef } from 'react';
+import { View, ViewProps } from 'react-native';
+import { ArbitrumIcon } from 'src/icons/ArbitrumIcon';
+import { EthIcon } from 'src/icons/EthIcon';
+import { OptimismIcon } from 'src/icons/OptimismIcon';
+import { PolygonIcon } from 'src/icons/PolygonIcon';
+import { SpinnerIcon } from 'src/icons/SpinnerIcon';
+import { ZoraIcon } from 'src/icons/ZoraIcon';
+
+import { BottomSheetRow } from '../Feed/Posts/PostBottomSheet';
+import {
+  GalleryBottomSheetModal,
+  GalleryBottomSheetModalType,
+} from '../GalleryBottomSheet/GalleryBottomSheetModal';
+import { useSafeAreaPadding } from '../SafeAreaViewWithPadding';
+import { Typography } from '../Typography';
+
+const SNAP_POINTS = ['CONTENT_HEIGHT'];
+
+type Props = {
+  onConnectWallet: () => void;
+  isSignedIn: boolean;
+};
+
+function WalletSelectorBottomSheet(
+  { onConnectWallet, isSignedIn }: Props,
+  ref: ForwardedRef<GalleryBottomSheetModalType>
+) {
+  const { bottom } = useSafeAreaPadding();
+
+  const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
+
+  const { animatedHandleHeight, animatedSnapPoints, animatedContentHeight, handleContentLayout } =
+    useBottomSheetDynamicSnapPoints(SNAP_POINTS);
+
+  return (
+    <GalleryBottomSheetModal
+      ref={(value) => {
+        bottomSheetRef.current = value;
+
+        if (typeof ref === 'function') {
+          ref(value);
+        } else if (ref) {
+          ref.current = value;
+        }
+      }}
+      snapPoints={animatedSnapPoints}
+      handleHeight={animatedHandleHeight}
+      contentHeight={animatedContentHeight}
+    >
+      <View
+        onLayout={handleContentLayout}
+        style={{ paddingBottom: bottom }}
+        className="p-4 flex flex-col space-y-6"
+      >
+        <View className="flex flex-col space-y-4">
+          {isSignedIn ? (
+            <Typography
+              className="text-lg text-black-900 dark:text-offWhite text-center"
+              font={{ family: 'ABCDiatype', weight: 'Bold' }}
+            >
+              Please sign the message in your wallet
+            </Typography>
+          ) : (
+            <Typography
+              className="text-lg text-black-900 dark:text-offWhite"
+              font={{ family: 'ABCDiatype', weight: 'Bold' }}
+            >
+              Network
+            </Typography>
+          )}
+        </View>
+
+        <View className="flex flex-col space-y-2">
+          {isSignedIn ? (
+            <View
+              className="items-center"
+              style={{
+                paddingBottom: bottom,
+              }}
+            >
+              <SpinnerIcon spin />
```

Comment:
looks like we  need a dark mode version 😅
https://github.com/gallery-so/gallery/assets/80802871/b5adf765-2c47-4d53-b253-e8ac03aa845c

Reply:
lololo. Updated on dark mode version 🤦‍♂️


https://github.com/gallery-so/gallery/assets/4480258/77606d7b-3840-4255-a3e7-8d5122e09e7f